docs/deploy-cephfs.md

```diff
@@ -148,7 +148,7 @@ for more information.
 **Deploy Ceph configuration ConfigMap for CSI pods:**
 
 ```bash
-kubectl create -f ../../../examples/ceph-conf.yaml
+kubectl create -f ../../../deploy/ceph-conf.yaml
```

Comment:
it is assumed you are in a subdir of the `deploy` directory already, you can remove a few `../` and `deploy/` from the path now.

Reply:
Argh.. Missed it in the previous update. Corrected now.